### PR TITLE
payment-links: Add payment links list

### PR DESCRIPTION
**Pull request by julianasobreira**
## Checklist
- [ ] Criar lista de payment list

## Issues linkadas
- [ ] Related to https://github.com/pagarme/pilot/issues/1516

### Layout:
image

### Preview:
image

## Como testar?
`$ yarn run storybook`

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: e8567ec9fde4d5521820af41d7b30967d787a0aa
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: e8567ec9fde4d5521820af41d7b30967d787a0aa
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: e8567ec9fde4d5521820af41d7b30967d787a0aa
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: e8567ec9fde4d5521820af41d7b30967d787a0aa
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: packages/pilot/stories/containers/PaymentLinksList/index.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import links from './links.json'
+
+const t = sentence => path(split('.', sentence), translations)
+
+const WithEmptyList = () => (
+  <Section>
+    <PaymentLinksList
+      t={t}
+      amount={0}
+      rows={[]} // conferir como é o retorno dos links
```
Comment (eduardogspereira) [inline packages/pilot/stories/containers/PaymentLinksList/index.js:17]: Esse comentario eh bom deixar? :)

File: packages/pilot/src/containers/PaymentLinks/PaymentLinksList/index.js [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+  >
+    <Message
+      image={<EmptyStateIcon width={365} height={148} />}
+      message={(
+        <Fragment>
+          <div>
+            {t('pages.transactions.no_results_title')}
+          </div>
+          <div>
+            {t('pages.transactions.no_results_message')}
```
Comment (eduardogspereira) [inline packages/pilot/src/containers/PaymentLinks/PaymentLinksList/index.js:63]: Talvez seja melhor colocar detnro de `pages.payment_links`?

File: packages/pilot/src/containers/PaymentLinks/PaymentLinksList/index.js [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+const buildEmptyState = t => (
+  <Flexbox
+    alignItems="center"
+    className={style.emptyStateBlock}
+    direction="column"
+  >
+    <Message
+      image={<EmptyStateIcon width={365} height={148} />}
+      message={(
+        <Fragment>
```
Comment (eduardogspereira) [inline packages/pilot/src/containers/PaymentLinks/PaymentLinksList/index.js:58]: O que acha de usar a outra sintaxe para o Fragment? `<>`

File: packages/pilot/src/containers/PaymentLinks/PaymentLinksList/style.css [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+    font-weight: 600;
+  }
+}
+
+.centralizedItem {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  & .helpIcon {
```
Comment (eduardogspereira) [inline packages/pilot/src/containers/PaymentLinks/PaymentLinksList/style.css:48]: Acho que isso aqui pode ser removido, ou nao?

File: packages/pilot/src/containers/PaymentLinks/PaymentLinksList/tableColumns.js [hunk 5]
```diff
@@ -0,0 +1,76 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
```
Comment (eduardogspereira) [inline packages/pilot/src/containers/PaymentLinks/PaymentLinksList/tableColumns.js:1]: Esse lint aqui da pra ser removido :)

File: packages/pilot/src/containers/PaymentLinks/PaymentLinksList/index.js [hunk 6]
```diff
@@ -0,0 +1,235 @@
...
+  onPageCountChange: PropTypes.func.isRequired,
+  onRowClick: PropTypes.func.isRequired,
+  onSelectRow: PropTypes.func.isRequired,
+  order: PropTypes.string,
+  orderField: PropTypes.arrayOf(PropTypes.string),
+  pagination: PropTypes.shape({
+    offset: PropTypes.number,
+    total: PropTypes.number,
+  }).isRequired,
+  query: PropTypes.shape({
```
Comment (eduardogspereira) [inline packages/pilot/src/containers/PaymentLinks/PaymentLinksList/index.js:207]: Ow ju, talvez seja melhor remover esse query e quando colocarmos o PaymentLinkFilter aqui a gente adiciona novamente.

**Pull request merged**